test/wt_byte_epr_test.cpp

```diff
@@ -0,0 +1,189 @@
+#include "sdsl/wavelet_trees.hpp"
+#include "common.hpp"
+#include "gtest/gtest.h"
+#include <vector>
+#include <string>
+#include <algorithm> // for std::min
+#include <random>
+
+namespace
```

Comment:
There's a `wt_byte_test`. Can you check if it works if you add it there to both Implementation sets (FULL_TEST_SUITE) ?